src/ruby/ext/grpc/rb_call_credentials.c

```diff
@@ -118,9 +119,9 @@ static int grpc_rb_call_credentials_plugin_get_metadata(
     grpc_metadata creds_md[GRPC_METADATA_CREDENTIALS_PLUGIN_SYNC_MAX],
     size_t* num_creds_md, grpc_status_code* status,
     const char** error_details) {
-  callback_params* params = gpr_malloc(sizeof(callback_params));
+  callback_params* params = gpr_zalloc(sizeof(callback_params));
```

Comment:
What's the difference here?

Reply:
The only difference is `gpr_zalloc` zeros out the contents of the allocated memory, which can be useful e.g. for crashing early in the case of memory corruption.